guide/designing-products/common-user-flows.md

```diff
@@ -413,7 +425,9 @@ Once a user has requested payment, they are naturally interested in knowing when
    layout = "float-right"
 %}
 
-A user may also want to check in and see if any previous requests have not been completed yet. This is easily possible if the user has initiated all requests on the same wallet and used a new address for each one. In this case, a request can be considered fulfilled if at least one payment has been received with the total amount the user asked for. It is not as clear if addressess are getting re-used (how to tell which payment was for which purpose?) or the request has been made with another wallet (as this meta data is not stored and synced via the Bitcoin network).
+A user may also want to check in and see if any previous requests have not been completed yet. This is easily possible if the user has initiated all requests on the same wallet and used a new address for each one. In this case, a request can be considered fulfilled if at least one payment has been received with the total amount the user asked for. It is not as clear if addresses are getting re-used (how to tell which payment was for which purpose?) or the request has been made with another wallet (as this meta data is not stored and synced via the Bitcoin network).
+
+On the Lightning network, receiving bitcoin requires an invoice. This makes it easy to track if payments have been completed or not.
```

Comment:
This might also change with keysend, Lightning addresses, and other techniques. The thing is that an invoice is still required, but the wallet does some under the hood communication with the receiving node, so the user never sees the invoice part.

Maybe we need to discuss how to go about this? Do we base our descriptions on the basic default Lightning implementation using invoices, or the UX you get with those extra techniques?

Reply:
@GBKS I am not sure that those other methods are going to affect UX in a way too much different than on-chain. As you describe, they all create invoices, but under-the-hood. From a user perspective, I think of this as being somewhat synonymous to having somebody send you money to a BTC address you have listed on your website: the money simply arrives without you having asked for it.

The primary UX concern I see for these techniques is "consent". For example, if a wallet is compatible with something like LNURL-pay, there might be an option in the app's settings menu along the lines of "Allow payments without asking" to toggle this feature on/off. (Perhaps some people don't want to receive money from people they don't know; most likely wouldn't care).

Reply:
@danielnordh My one comment, and this applies to other mentions of LN in this section of the Guide: should we move sentences about LN to the top of their sections and not have them at the bottom? I feel like the LN content is kind of tucked away at the bottom. So for example, instead of "Receiving Bitcoin" going into depth about on-chain and then having a brief mention of LN at the bottom, perhaps LN is mentioned prior to the paragraphs about receiving on-chain?

Not critical, but it would help to elevate the LN content more.

Reply:
I see your point @sbddesign, but personally I would find it a bit weird if Lightning is mentioned before the base layer in most of these sections. Maybe this changes at a later state of the guide.